docs/http/jwt-validation.mdx

```diff
@@ -0,0 +1,106 @@
+:::
+
+<Tabs groupId="connectivity" queryString="cty">
+	<TabItem value="agent-cli" label="Agent CLI" default>
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="agent-config" label="Agent Config">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="ssh" label="SSH">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="go-sdk" label="Go">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="javascript-sdk" label="Javascript">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="python-sdk" label="Python">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="rust-sdk" label="Rust">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="k8s" label="Kubernetes Controller">
+		Currently Unavailable
+	</TabItem>
+</Tabs>
+
+## Behavior
+
+### Multiple Issuers
+
+You may specify multiple Issuers to be used for JWT validation. A request
+is considered validated if it presents a JWT signed by any of the
+specified issuers.
+
+### Multiple Audience Claims
+
+You may specify multiple audience claims to be used for JWT validation. JWT validation will require one of the audience claims to be present within the JWT.
+
+### Multiple JWS
+
+You have the ability to provide multiple JWKS urls. During JWT validation the list of JWKS provided will be used to validate the JWT.
+
+### Multiple Tokens
+
+If multiple tokens are defined within the HTTP configuration parameter, all tokens must be present in the request. I all tokens are not present a 401 status code will be returned.
+
+## Reference
+
+### Configuration
+
+| Parameter    | Description                                                                                                     |
+| ------------ | --------------------------------------------------------------------------------------------------------------- |
+| **issuer**   | A list of allowed issuers.                                                                                      |
+| **audience** | A list of allowed aud claims.                                                                                   |
+| **http**     | Configurations for handling JWT within a requests. Options include type of JWT and location within the request. |
+| **jws**      | Configurations defining how the key was signed. Options include signing algorithms and url for the jwks.        |
+
+### Upstream Headers {#upstream-headers}
+
+This module does not add any upstream headers.
+
+### Errors
+
+| Code                | HTTP Status | Error                                                                |
+| ------------------- | ----------- | -------------------------------------------------------------------- |
+| no ngrok error code | `401`       | This error is returned if the request is unauthorized.               |
+| no ngrok error code | `403`       | This error is returned if the JWT cannot be validated by the issuer. |
```

Comment:
I believe we have several error codes for various different problems: https://github.com/ngrok-private/ngrok/blob/main/config/src/errors/mw.ts#L1019

Reply:
@wdawson Do those errors get exposes to end users?  In my testing I've only been able to raise 3401 and 3403.

Reply:
ah! looks like no. we can keep that for security or change it to balance more towards devex I think